chapter_linear-networks/linear-regression.md

```diff
@@ -38,14 +38,14 @@ $$\hat{y} = \mathbf{w}^\top \mathbf{x} + b.$$
 
 在 :eqref:`eq_linreg-y` 中，向量 $\mathbf{x}$ 对应于单个数据样本的特征。用符号表示的矩阵 $\mathbf{X} \in \mathbb{R}^{n \times d}$ 可以很方便地引用我们整个数据集的 $n$ 个样本。其中，$\mathbf{X}$ 的每一行是一个样本，每一列是一种特征。
 
-对于特征集合 $\mathbf{X}$ 和预测值 $\hat{\mathbf{y}} \in \mathbb{R}^n$ 可以通过矩阵-向量乘法表示为：
+对于特征集合 $\mathbf{X}$ ，预测值 $\hat{\mathbf{y}} \in \mathbb{R}^n$ 可以通过矩阵-向量乘法表示为：
 
 $${\hat{\mathbf{y}}} = \mathbf{X} \mathbf{w} + b$$
 
-这个过程的求和过程将使用广播机制。广播机制会在  :numref:`subsec_broadcasting`)详细介绍。
+这个过程中的求和将使用广播机制(广播机制在  :numref:`subsec_broadcasting` 中有详细介绍)。
```

Comment:
Please use brackets in Chinese. 
"（", not "("

Reply:
done in new commit.